bin/setup

```diff
@@ -26,6 +26,9 @@ Dir.chdir APP_ROOT do
   puts "== Copying application.yml =="
   run "cp config/application.yml.example config/application.yml"
 
+  puts "== Copying logstash.conf =="
+  run "cp logstash.conf.example logstash.conf"
```

Comment:
Can we have this script replace `path_to_repo` ? Would simplify the README

Ex

```
run "cat logstash.conf.example | sed 's/path_to_repo/#{APP_ROOT}/g > logstash.conf"
```